### DISCUSSION TITLE
Base table or view not found: 1146 Table 'tribe.abilities' doesn't exist

### DISCUSSION THREAD

**Discussion by Temian1** [filamentphp/filament#2674] [Help]
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'tribe.abilities' doesn't exist

i got this error when i created post resources i dont now why but how can i solve this

**Comment by ryangjchandler** [comment]
This table isn't created or related to Filament, this is likely a package that you're using or one of your own tables.
